<launch>

  <arg name="name" default="husky"/>

  <!-- Settings for ATAK communications -->
  <arg name="callsign" default="$(arg name)" doc="The callsign used by the robot on the ATAK system"/>
  <arg name="team_name" default="Ground Robots" doc="The Team Name used by the robot on the ATAK system"/>
  <arg name="team_role" default="Autonomous Agent" doc="The Team Role used by the robot on the ATAK system"/>
  <arg name="tak_ip" default="10.13.0.10" doc="The IP address of the ATAK server"/>
  <arg name="tak_port" default="8088" doc="The Port number on the ATAK server to connect to"/>
  <arg name="baselink_frame" default="/husky/base_link" doc="The base link frame of the robot"/>
  <arg name="global_frame" default="husky/odom" doc="The global coordinate frame used to send global locations to ATAK"/>

<!-- 
41.389974, -73.955016 37 degrees magnetic TH1136 
41.39058,  -73.95323  0  degrees magnetic River Courts
-->

  <group ns="$(arg name)/atak">
  
    <node name="dcist_cots" pkg="atak_bridge" type="oop_dcist_grnd.py" output="screen">
<!--    <node name="dcist_cots" pkg="atak_bridge" type="obj_2_atak_uav.py" output="screen">-->
        <param name="name" value="$(arg name)" />
        <param name="callsign" value="$(arg callsign)" />
        <param name="team_name" value="$(arg team_name)" />
        <param name="team_role" value="$(arg team_role)" />
        <param name="tak_ip" value="$(arg tak_ip)" />
        <param name="tak_port" value="$(arg tak_port)" />
        <param name="baselink_frame" value="$(arg baselink_frame)" />
        <param name="global_frame" value="$(arg global_frame)" />
    </node>

  </group> 

</launch>

